packages/data/src/atomic-store/index.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * External dependencies
+ */
+import { mapValues } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { createDerivedAtom } from '@wordpress/stan';
+
+/**
+ *
+ * @param {string}                                     name   Store name.
+ * @param {import('../types').WPDataAtomicStoreConfig} config Atomic store config.
```

Comment:
A similar issue I had in #27061. There needs to be a `typedef` to ensure that auto-generated documentation works correctly, otherwise we see `(unknown type)` instead of  `WPDataAtomicStoreConfig`.